sanctuarycomputer/contentful-fragment
# Convert UTC to local when rendering date field

Fixes https://github.com/sanctuarycomputer/contentful-fragment/issues/20

To test: 
Change `DEV=true` in application.js
`yarn start`
Edit contentful fragment, and select a date using the date picker. Then click the check mark to persist the data. Confirm that the date you selected is the same date that is displayed in the saved fragment card.
